<ui version="4.0" >
 <class>prefs_projectmain</class>
 <widget class="QWidget" name="prefs_projectmain" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>292</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="KEditListBox" name="kcfg_WebQueryScripts" >
     <property name="title" >
      <string>GroupBox</string>
     </property>
    </widget>
   </item>
  </layout>
  <widget class="RelPathSaver" name="_GlossaryTbx" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>267</y>
     <width>480</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>RelPathSaver</class>
   <extends>QLineEdit</extends>
   <header>relpathsaver.h</header>
  </customwidget>
  <customwidget>
   <class>KEditListBox</class>
   <extends>QGroupBox</extends>
   <header location="global" >keditlistbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
